java/code/src/com/redhat/rhn/domain/contentmgmt/test/ContentFilterTest.java

```diff
@@ -48,13 +48,13 @@ public void setUp() throws Exception {
         UserTestUtils.addUserRole(user, ORG_ADMIN);
     }
 
-    public void testPackageDenyFilter() throws Exception {
+    public void testPackageFilter() throws Exception {
         Package pack = PackageTest.createTestPackage(user.getOrg());
         String packageName = pack.getPackageName().getName();
 
         FilterCriteria criteria = new FilterCriteria(FilterCriteria.Matcher.CONTAINS, "name", packageName);
         ContentFilter filter = ContentManager.createFilter(packageName + "-filter", DENY, PACKAGE, criteria, user);
-        assertFalse(filter.test(pack));
+        assertTrue(filter.test(pack));
```

Comment:
test() does not depend anymore on ALLOW or DENY. It is just a criteria match or not.
ALLOW and DENY is evaluated later. Do I get it right?

Reply:
Exactly.